bittensor/utils/dispatcher.py

```diff
@@ -113,6 +113,9 @@ def combine(self, expert_out, gates, multiply_by_gates=True):
         combined = zeros.index_add(0, batch_index, flat_stitched.float())
 
         # reshape as original
-        combined = combined.view(expert_out[0].shape)
+        try:
+          combined = combined.view(expert_out[0].shape)
+        except:
+          pass
```

Comment:
this is dangerous. Should at least give a logger error if we don't want to raise here.

Reply:
This shouldn't be here.  Thanks